### ISSUE TITLE
What does batchs_per_iter do 

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#150]
What is the difference between batch_per_iter and batch_size? What does batchs_per_iter do ?

**Comment by Daniil-Osokin** [comment]
Hi! The good batch should approximate training data distribution well. So bigger batch is usually better for that. Different gpus have different amounts of memory: some allow bigger batches, another smaller ones. So to emulate the effect of bigger batch with small memory gpu, you can set `batch_per_iter > 1` to accumulate gradients from multiple batches. So weights will be updated with averaged gradients from `batch_per_iter` batches.

**Comment by tangfayuan** [comment]
thanks for your help
